.github/workflows/build.yaml

```diff
@@ -39,19 +42,25 @@ jobs:
     needs: test
     steps:
     - uses: actions/checkout@v2
+      with:
+        # Fetch all history instead of the latest commit
+        fetch-depth: 0
     - name: Set up Python
       uses: actions/setup-python@v2
       with:
         python-version: 3.8
+    - name: Remove version file
+      # This is just in case something else created it, destroy it to get a fresh version
+      run: rm -f dysql/version.py
     - name: Install wheel
       run: pip install wheel
     - name: Build
       run: python setup.py sdist bdist_wheel
     - name: Check upload
       run: pip install twine && twine check dist/*
     - name: Publish to PyPi
-      # Only publish if this is a tag
-      if: github.event_name == 'push' && startsWith(github.ref, 'refs/tags')
+      # Only publish on pushes to main
```

Comment:
does this make it so we publish main anytime something gets pushed to it? I mean, we should always be using PRs but I'm just curious

Reply:
yup